backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/calendar/internal/BpkCalendarDayCell.kt

```diff
@@ -194,8 +194,7 @@ private fun Modifier.highlightedDayBackground(
     highlighted: Boolean,
 ): Modifier =
     if (highlighted) this
-        .border(2.dp, coreAccent, CircleShape)
-        .padding(10.dp) else this
```

Comment:
I thought the padding was added to ensure there’s some space between the magnifying glass and the circle. If we remove it entirely, we might run into the same issue we had before. Ideally, we should find a solution that maintains spacing between the label (icon, text) and the highlighted cell when space is available, but removes the padding when it’s not.

Reply:
checking the code again, this padding is redundant as we have the cell components wrapped in a box meaning they are drawn on top of each other -> padding is not going to push the magnified glass bellow anyway.
|not selected|selected|
|-|-|
![image](https://github.com/user-attachments/assets/bedc0e02-0471-40a0-b0ab-6df4a17d8dba)|![image](https://github.com/user-attachments/assets/d447a51e-de7b-49da-b85f-4f73922945ff)

Reply:
this looks fine to me